<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuCrearPartida</class>
 <widget class="QMainWindow" name="MenuCrearPartida">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Worms 2D - Menu de partidas</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>640</width>
     <height>480</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>640</width>
        <height>480</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="nuevaPartidaButton">
       <property name="geometry">
        <rect>
         <x>158</x>
         <y>400</y>
         <width>150</width>
         <height>40</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>300</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: white;
	border-style: outset;
	border-width: 2px;
	border-radius: 10px;
	font: bold 14px;
}

QPushButton:hover {
	background: grey;
}

QPushButton:pressed {
	background-color: grey;
	border-style: inset;
}</string>
       </property>
       <property name="text">
        <string>Crear partida</string>
       </property>
      </widget>
      <widget class="QComboBox" name="jugadoresComboBox">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>200</y>
         <width>180</width>
         <height>40</height>
        </rect>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::DefaultContextMenu</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QComboBox {
	background-color: white;
	border-style: outset;
	border-width: 2px;
	border-radius: 10px;
	font: bold 14px;
}

QComboBox QAbstractItemView {
	background: white;
	background-color: darkgray;
	border-style: outset;
	border-width: 2px;
	border-color: grey;
	font: bold 14px;
	color: black;
	border: 2px solid darkgray;
}

QComboBox::drop-down {
	border-left-width: 1px;
	border-left-color: darkgray;
	border-left-style: solid;
	border-top-right-radius: 3px;
	border-bottom-right-radius: 3px;
}

QComboBox::item:hover {
	color: black;
	background-color: gray;
}</string>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
       <property name="currentText">
        <string>Seleccione una opcion</string>
       </property>
       <item>
        <property name="text">
         <string>Seleccione una opcion</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="mapaComboBox">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>250</y>
         <width>180</width>
         <height>40</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QComboBox {
	background-color: white;
	border-style: outset;
	border-width: 2px;
	border-radius: 10px;
	font: bold 14px;
}

QComboBox QAbstractItemView {
	background: white;
	background-color: darkgray;
	border-style: outset;
	border-width: 2px;
	border-color: grey;
	font: bold 14px;
	color: black;
	border: 2px solid darkgray;
}

QComboBox::drop-down {
	border-left-width: 1px;
	border-left-color: darkgray;
	border-left-style: solid;
	border-top-right-radius: 3px;
	border-bottom-right-radius: 3px;
}

QComboBox::item:hover {
	color: black;
	background-color: gray;
}</string>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
       <property name="currentText">
        <string>Seleccione una opcion</string>
       </property>
       <item>
        <property name="text">
         <string>Seleccione una opcion</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="volverAtrasButton">
       <property name="geometry">
        <rect>
         <x>332</x>
         <y>400</y>
         <width>150</width>
         <height>40</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>300</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: white;
	border-style: outset;
	border-width: 2px;
	border-radius: 10px;
	font: bold 14px;
}

QPushButton:hover {
	background: grey;
}

QPushButton:pressed {
	background-color: grey;
	border-style: inset;
}</string>
       </property>
       <property name="text">
        <string>Volver atras</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelJugadores">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>210</y>
         <width>161</width>
         <height>17</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Cantidad de jugadores</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelMapa">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>260</y>
         <width>41</width>
         <height>17</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Mapa</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelPartida">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>160</y>
         <width>91</width>
         <height>17</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>ID de partida</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="partidaLineEdit">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>150</y>
         <width>180</width>
         <height>40</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
	background-color: white;
	border-style: outset;
	border-width: 2px;
	border-radius: 10px;
	font: bold 14px;
}

QLineEdit QAbstractItemView {
	background: white;
	background-color: darkgray;
	border-style: outset;
	border-width: 2px;
	border-color: grey;
	font: bold 14px;
	color: black;
	border: 2px solid darkgray;
}
</string>
       </property>
       <property name="text">
        <string>Ingrese un ID</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
